compiler/optimizer/Dominators.hpp

```diff
@@ -53,11 +53,14 @@ class TR_Dominators
    TR_ALLOC(TR_Memory::Dominators)
 
    TR_Dominators(TR::Compilation *, bool post = false);
+   TR_Dominators(TR::Compilation *, TR::CFG* cfg, bool acceptUnreachableBlocks = true, bool post = false);
```

Comment:
Does `acceptUnreachableBlocks` need to be a parameter? I expect `TR_RegionAnalysis::getRegions(...cfg...)` to be the only use of this. It's a parameter there as well, but I think it probably doesn't need to be